samples/nrf9160/secure_services/src/main.c

```diff
@@ -72,6 +72,10 @@ void main(void)
 	}
 
 	print_hex_number(buf, num_bytes_to_read);
+#elif defined(PM_S0_ADDRESS)
```

Comment:
Why elif? In case other code changes, I think it is best to have a "self contained" check here.

Reply:
This currently doesn't work when MCUBoot is included. I can change it to 

    #if defined(PM_S0_ADDRESS) && !defined(CONFIG_BOOTLOADER_MCUBOOT)

Reply:
I would prefer that, since it does not depend on the code around it.